### PR TITLE
Update frpc.toml, stop.sh, config.json, and start.sh

### PR DESCRIPTION
**Pull request by longzhi**

### CODE REVIEW

File: demo/frpc.toml [hunk 1]
```diff
@@ -2,9 +2,8 @@ serverAddr = "52.77.216.244"
 serverPort = 7000
 
 [[proxies]]
-name = "gaianet"
+name = "demo.gaianet.xyz"
```
Comment (juntao) [inline demo/frpc.toml:5]: The script will change the subdomain to something other than `demo`. Does this name stay as `demo`?

File: demo/init.sh [hunk 2]
```diff
@@ -309,20 +309,41 @@ else
...
+    exit 1
+fi
+
+# Replace the serverAddr in frpc.toml
+sed -i '' "s/serverAddr = \".*\"/serverAddr = \"$ip_address\"/g" $gaianet_base_dir/gaianet-domain/frpc.toml
+
+# Copy frpc.toml to dashboard/
+cp $gaianet_base_dir/gaianet-domain/frpc.toml $gaianet_base_dir/dashboard/
+
+printf "The subdomain for frpc is: http://$subdomain.$gaianet_domain:8080\n"
```
Comment (juntao) [inline demo/init.sh:346]: Let's get a TLS cert on our domain server and make this into `https`

**Review by juntao — APPROVED** [review]
While I still have minor questions (see review comments @longzhi ), I will merge this now so that we can build on these new features.

File: demo/frpc.toml (hunk 1 above)
Comment (longzhi) [reply demo/frpc.toml:5]: OK

**Pull request merged**